gsa/src/gmp/utils/calculator.js

```diff
@@ -0,0 +1,676 @@
+/* Copyright (C) 2017-2020 Greenbone Networks GmbH
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Affero General Public License
+ * as published by the Free Software Foundation, either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import {isDefined} from 'gmp/utils/identity';
+
+const toFixed1 = value => {
+  let result = +(
+    Math.round(+(value.toString() + 'e' + 1)).toString() +
+    'e' +
+    -1
+  );
+  // if no fractional part then add a .0 to the number
+  if (result.toString().indexOf('.') === -1) result = result + '.0';
```

Comment:
Please always use curly braces for if clauses. It increases the readability at lot.
```suggestion
  if (!`${result}`.includes('.')) {
    result = result + '.0';
  }
```

Reply:
IMHO using includes instead of indexOf is also better for the readability https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes